src/Redistribution.sol

```diff
@@ -636,6 +648,8 @@ contract Redistribution is AccessControl, Pausable {
 
         PostageContract.withdraw(winner.owner);
 
+        emit PostageStampStatus(PostageContract.validChunkCount(), PostageContract.pot());
```

Comment:
It should be before the withdraw operation since the pot will be `0` always.

Reply:
good point, this wasn't really needed. just added it as it could be useful. will remove it as we wanted to get validchunk after withdrawl and clearing of stamps